jeffjxu/Wesley-Central-Access-master
# Dev

change SCA layout and fields

how to test:
- check that the SCA form dropdowns are consistent with Aaron's image
- do the same for create, edit, and detail page
